torch/library.py

```diff
@@ -916,6 +920,7 @@ def register_fake(
     *,
     lib: Optional[Library] = None,
     _stacklevel: int = 1,
+    allow_override: bool = False,
```

Comment:
1) should document allow_override.
2) It turns out if `op` is a customopdef (e.g. created from torch.library.custom_op), we already allow overriding it by default. So I guess the allow_override flag only applies to non-custom-opdef

I'm not sure what the best way to handle this is. Maybe we just document it and move on. "allow_override: Only applies if the custom operator was not created via torch.library.custom_op. If we should allow overriding it..."